src/app/control-planes/views/ControlPlaneDetailView.vue

```diff
@@ -13,111 +13,166 @@
         </h1>
       </template>
 
-      <div
-        class="stack"
-        data-testid="detail-view-details"
+      <DataSource
+        v-slot="{ data: globalInsight, error: globalInsightError }: GlobalInsightSource"
+        src="/global-insight"
       >
-        <ControlPlaneStatus />
-
-        <div class="columns">
-          <KCard v-if="can('use zones')">
-            <template #body>
-              <DataSource
-                v-slot="{ data, error }: ZoneOverviewCollectionSource"
-                src="/zone-cps?page=1&size=10"
+        <DataSource
+          v-slot="{ data: meshInsightsData, error: meshInsightsError }: MeshInsightCollectionSource"
+          src="/mesh-insights?page=1&size=10"
+        >
+          <DataSource
+            v-slot="{ data: zoneOverviewsData, error: zoneOverviewsError }: ZoneOverviewCollectionSource"
+            src="/zone-cps?page=1&size=10"
```

Comment:
Does this mean we are loading zones data even if they aren't enabled?

Reply:
Hm, yes, that’s correct and a bug. I moved the zone data source up here because having two of them here and the third deeper in was strange. I might just move it back.

Reply:
This is fixed now. I moved the data sources back to where they were.